Gemfile

```diff
@@ -5,6 +5,7 @@ eval_gemfile(File.expand_path("Gemfile.common", __dir__))
 gem "rails", "~> 5.2.x"
 gem "devise", "~> 4.4"
 gem "draper", "~> 3.0"
+gem "activemodel-serializers-xml"
```

Comment:
Since we are adding a runtime dependency in the gemspec, we no longer need this addition to the Gemfile.